content/en/docs/tasks/job/indexed-parallel-processing-static.md

```diff
@@ -66,13 +66,21 @@ program accepts a file as an argument and prints its content reversed.
 rev data.txt
 ```
 
-For this example, you'll use the `rev` tool from the
+You'll use the `rev` tool from the
 [`busybox`](https://hub.docker.com/_/busybox) container image.
 
+As this is only an example, each Pod only does a tiny piece of work (reversing a short
+string). In a real workload you might, for example, create a Job that represents
+ the
+task of producing 60 seconds of video based on scene data.
+Each work item in the video rendering Job would be to render a particular
```

Comment:
nit: reword
Each work item in the video rendering Job renders a particular frame ...